src/JWT/Compatibility/String.cs

```diff
@@ -1,8 +1,16 @@
-﻿#if NET35
+#if NET35
+using System;
+
 namespace JWT.Compatibility
 {
     internal static class String
     {
+        public int IndexOf(string value, char ch, StringComparison comparisonType)
+        {
+            // ignores comparisonType since the overload is missing in .NET 3.5
+            return value.IndexOf(ch);
```

Comment:
Would just ignoring `comparisonType` be ok? Or it's better to use [the another overload](https://docs.microsoft.com/en-us/dotnet/api/system.string.indexof?view=netframework-3.5#System_String_IndexOf_System_String_System_StringComparison_):

```c#
public int IndexOf (string value, StringComparison comparisonType);
```
i.e.:

```c#
return value.IndexOf(ch.ToString(), comparisonType);
```

Reply:
I give up trying to make this working (